src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/FilterDelegationBase.cs

```diff
@@ -184,6 +184,19 @@ protected bool IsValidDelegatableFilterPredicateNode(TexlNode dsNode, TexlBindin
             return true;
         }
 
+        public override void CheckSemantics(TexlBinding binding, TexlNode[] args, DType[] argTypes, IErrorContainer errors)
```

Comment:
>override [](http://example.com/codeflow?start=15&length=8)

Putting on this base class make sense. 

It looks like this will catch: 
CountIf, Filter, LookUp.  (and possibly anything in PA derived that's not in Fx).